types/ember__test-helpers/ember__test-helpers-tests.ts

```diff
@@ -53,7 +53,11 @@ test('DOM interactions', async () => {
     await focus(messageElement);
     await blur(messageElement);
     await triggerEvent(messageElement, 'custom-event');
+    await triggerEvent(window, 'custom-event');
+    await triggerEvent(document, 'custom-event');
```

Comment:
IMHO, since this group is all testing passing an element, testing `window` and `document` should each be moved into a separate sections below:

```js
.
.
.
await fillIn(messageElement, 'content');
await typeIn(messageElement, 'content');

await triggerEvent(window, 'custom-event');
await triggerKeyEvent(window, 'keydown', 'Enter', { ctrlKey: true });

await triggerEvent(document, 'custom-event');
await triggerKeyEvent(document, 'keydown', 'Enter', { ctrlKey: true });
```

Reply:
Haha - I've seen this, there were two ways of doing it. The other I  had in mind was the one you suggested now =)